<ui version="4.0" >
 <class>editPBDialog</class>
 <widget class="QDialog" name="editPBDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Editing PKGBUILD</string>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>20</y>
     <width>411</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="editingLabel" >
        <property name="text" >
         <string>Editing PKGBUILD of:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="packagesBox" />
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QPushButton" name="saveButton" >
        <property name="text" >
         <string>Save</string>
        </property>
        <property name="icon" >
         <iconset resource="qtpacman.qrc" >:/Icons/icons/document-open.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="undoButton" >
        <property name="text" >
         <string>Undo</string>
        </property>
        <property name="icon" >
         <iconset resource="qtpacman.qrc" >:/Icons/icons/edit-delete.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="redoButton" >
        <property name="text" >
         <string>Redo</string>
        </property>
        <property name="icon" >
         <iconset resource="qtpacman.qrc" >:/Icons/icons/edit-redo.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="PBEdit" >
      <property name="lineWrapMode" >
       <enum>QTextEdit::NoWrap</enum>
      </property>
      <property name="acceptRichText" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton" >
        <property name="text" >
         <string>Ok</string>
        </property>
        <property name="icon" >
         <iconset resource="qtpacman.qrc" >:/Icons/icons/dialog-ok-apply.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="qtpacman.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>redoButton</sender>
   <signal>clicked()</signal>
   <receiver>PBEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>380</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel" >
     <x>225</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>undoButton</sender>
   <signal>clicked()</signal>
   <receiver>PBEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>279</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel" >
     <x>225</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
